pyproject.toml

```diff
@@ -16,20 +16,15 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 
-packages = [{ include = "infrahub", from = "backend" }]
-
-
-[tool.poetry.group.test-scale]
-optional = true
+packages = [
+    { include = "infrahub", from = "backend" },
+    { include = "infrahub_sdk", from = "python_sdk" }
```

Comment:
I'm wondering how this will behave on systems where where we want to install both the SDK and infrahub-server if one of them overwrites the other. It should mostly be relevant for testing purposes like when we want to install `infrahub-server` within the SDK repo during CI. I.e if the `infrahub_sdk` folder gets overwritten by the latest package of what happens.

Reply:
Yes for now there will be a conflict but I think the risk is limited for now.
it will take a few more steps to fix that, I've opened a new PR earlier today in the SDK repo that will get us closer https://github.com/opsmill/infrahub-sdk-python/pull/80